tutorials/thermal_camera.md

```diff
@@ -0,0 +1,268 @@
+\page thermalcamera Thermal Camera
+
+In this tutorial, we will discuss how to use a thermal camera sensor.
```

Comment:
A high level comment about having this tutorial in ign-sensors - Since ign-sensors (like other ignition libraries) is designed to be a standalone library that can be used outside of ign-gazebo, we should expect that in the future there maybe users outside of ign-gazebo reading these tutorials who wish to know how to use the thermal camera through the provided APIs. In this case, this tutorial is not for them. I think we should be more explicit that this tutorial is targeted at ign-gazebo users instead. 

We could make it clear in this first sentence to say `... use a thermal camera sensor through Ignition Gazebo`, and in addition, update the heading of this tutorial. Optionally, we may also want to state some of prerequisites to this tutorial like we assume users have ignition-gazebo3 / citadel or above installed (blueprint does come with thermal camera sensor), and if not we can point them to the ign-gazebo installation page.

Reply:
Those are good points, thanks for the reminder! I forgot that users may not be familiar with (or may not be using) Ignition Gazebo in their projects. I went ahead and tried to clarify the dependency on Ignition Gazebo in https://github.com/ignitionrobotics/ign-sensors/pull/61/commits/eec1461e01132e85b64cb208664a7f9ce0cddd75, as well as providing links to things like `SDF` in case users aren't aware of that either.